recipes/mbits-args/all/conanfile.py

```diff
@@ -1,103 +1,122 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import (
+    apply_conandata_patches,
+    export_conandata_patches,
+    get,
+    copy,
+    rmdir,
+)
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
 import os
 
 
+required_conan_version = ">=1.53.0"
+
+
 class MBitsArgsConan(ConanFile):
     name = "mbits-args"
-    description = "Small open-source library for program argument parser, inspired by Python's `argparse`, " \
-                  "depending only on the standard library, with C++17 as minimum requirement."
-    homepage = "https://github.com/mbits-libs/args"
+    description = (
+        "Small open-source library for program argument parser, inspired by Python's `argparse`, "
+        "depending only on the standard library, with C++17 as minimum requirement."
+    )
     license = "MIT"
-    topics = ("command-line", "commandline", "commandline-interface",
-              "program-arguments", "argparse", "argparser", "argument-parsing")
-
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mbits-libs/args"
+    topics = (
+        "command-line",
+        "commandline",
+        "commandline-interface",
+        "program-arguments",
+        "argparse",
+        "argparser",
+        "argument-parsing",
+    )
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-    generators = "cmake"
-    exports_sources = "CMakeLists.txt"
-
-    _cmake = None
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
```

Comment:
Just as a future reference, we like to deprecate options before completely removing this to make it smoother for consumers 😄 

The shared here should have been kept even though it's modified :)